pandas/io/gbq.py

```diff
@@ -46,8 +46,8 @@ def _test_google_api_imports():
 
     try:
         import httplib2  # noqa
-        from apiclient.discovery import build  # noqa
```

Comment:
I don't know the detail of GBQ, but does it breaks users who use ``apiclient`` now? Can there be any fallback logic?